src/contents/all/components/app/LoginForm.tsx

```diff
@@ -0,0 +1,101 @@
+/*global chrome*/
+
+import React, { useEffect, useState } from 'react';
+import Frame, { FrameContextConsumer } from 'react-frame-component';
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from 'react-router-dom';
+import { hot } from 'react-hot-loader/root';
+import * as Sentry from '@sentry/browser';
+
+import { Form, Icon, Input, Button, message, notification } from 'antd';
+
+import './App.scss';
+
+//import './LoginForm.scss'
+
+interface Props {
+  form: any
+}
+
+const FormItem = Form.Item;
+
+const LoginForm = () => {
+
+  const {error, inProgress} = useSelector(state => ({ ...state.auth, ...state.common}))
+
+  const submitForm = ev => {
+    ev.preventDefault();
+    form.validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        chrome.runtime.sendMessage({
+          message: "auth_user",
+          user: {
+            email: values.email,
+            password: values.password
+          }
+        })
+      }
+    })
+  }
+
+  useEffect(()=>{
+    if (error) {
+      if (error instanceof String) {
+        message.error(error)
+      } else {
+        for (const [ key, value ] of Object.entries(error)) {
+          message.error(`${key.charAt(0).toUpperCase()}${key.slice(1)} ${value}`)
+        }
+      }
+    }
+  },[inProgress])
+
```

Comment:
@spencerjsmall This also is because of the redux in mainframe. So not necessary here as we can use a local state to keep track of loading and when we get a response to update the errors if there are any (like wrong password etc)